x-pack/plugin/esql/qa/testFixtures/src/main/resources/range.csv-spec

```diff
@@ -0,0 +1,52 @@
+Invalid boundaries integer literal
+
+FROM employees 
+| WHERE salary > 200 AND salary < 100 
+| KEEP salary;
+
+salary:integer
+;
+
+Invalid boundaries date 
+
+FROM employees
+| WHERE hire_date > TO_DATETIME("2025-01-01") AND hire_date < TO_DATETIME("2020-01-01")
+| KEEP hire_date;
+
+hire_date:datetime
+;
+
+Invalid boundaries, date with implicit casting
+
+FROM employees
+| WHERE hire_date > "2025-01-01" AND hire_date < "2020-01-01"
+| KEEP hire_date;
+
+hire_date:datetime
+;
+
+Invalid Boundaries, date nanos 
+
+FROM date_nanos
+| WHERE nanos > TO_DATE_NANOS("2025-01-01") and nanos < TO_DATE_NANOS("2020-01-01")
```

Comment:
Because we use date nanos, I think at least the backport will require a capability to avoid broken bwc tests?